app/controllers/album.controller.js

```diff
@@ -45,14 +46,47 @@ exports.findAll = (req, res) => {
     });
 };
 
-exports.findOne = (req, res) => {
-  const id = req.params.id;
-  return Album.findByPk(id, { include: ["artist"] })
-    .then((album) => {
-      res.status(200).send(album)
+exports.findOne = async (req, res) => {
+  await getSingleAlbum(req.params.id).then((data) => {
+      res.status(200).send(data);
       return album;
     })
     .catch((err) => {
       console.log(">> Error while finding album: ", err);
     });
-};
\ No newline at end of file
+};
+
+
+function getSingleAlbum(id) {
+  return  Album.findByPk(id, { include: ["artist"] })
+}
+
+//update album name
+exports.editAlbumName = async (req, res) => {
```

Comment:
can we call it editAlbumTitle()? just to be consistent, we are only updating the album title (field)not the name.